vagen/envs/webarena/browser_env/actions.py

```
@@ -0,0 +1,2006 @@
+        case "click":
+            return create_click_action(pw_code=playwright_code)
+        case "clear":
+            return create_clear_action(pw_code=playwright_code)
+        case "hover":
+            return create_hover_action(pw_code=playwright_code)
+        case "type" | "fill":
+            p = r'type|fill\((?:"|\')(.+?)(?:"|\')\)'
+            match = re.search(p, playwright_code)
+            if not match:
+                raise ActionParsingError(
+                    f"Invalid type/fill action, required to be page.type(TEXT)"
```

Ruff (0.15.10)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)